roles/docker-influxdb-grafana/files/nginx_config/telegraf_nginx.conf

```diff
@@ -0,0 +1,34 @@
+server {
+        listen 443 ssl;
+
+        server_name telegraf.outcatcher.com;
+
+        proxy_http_version 1.1;
+        chunked_transfer_encoding off;
+
+        ##
+        # SSL Settings
+        ##
+
+        ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+        ssl_prefer_server_ciphers on;
+
+        ssl_certificate /etc/sslcerts/live/fullchain.pem;
+        ssl_certificate_key /etc/sslcerts/live/privkey.pem;
```

Comment:
Maybe this can be moved `http` section of parent config as duplicating

Reply:
Checked with default config, unsuccessful. Seems that need to specify ssl settings for every servers